src/components/bottom-bar/_bottom-bar.scss

```diff
@@ -80,7 +80,7 @@
 }
 
 .player.casting,
-.player.metadata-loaded.hover,
+.player.metadata-loaded:hover,
```

Comment:
1. the player has a timeout for hover toggling. see https://github.com/kaltura/playkit-js-ui/blob/master/src/components/shell/shell.js#L322
it won't work with this change
2. please note the top-bar has these css rulls too